client/src/cli/apiToken.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  getAccessToken,
+  CantCreateRmmbrDir,
+  NoAccessToken,
+  NoHomeDirectory,
+} from "./configDir.ts";
+import { Result, ok, err, Unit } from "./deps.ts";
+
+class InvalidAccessToken extends Error {}
+
+export async function apiToken(): Promise<Result<string, Unit>> {
+  const accessToken = await getAccessToken();
+  if (accessToken.isErr) {
+    if (accessToken.error instanceof CantCreateRmmbrDir) {
+      console.error("Couldn't create the `.rmmbr` directory.");
+    } else if (accessToken.error instanceof NoHomeDirectory) {
+      console.error("Couldn't find your home directory.");
+    } else if (accessToken.error instanceof NoAccessToken) {
```

Comment:
propagte the error, don't rewrite it, so the log text is already included in NoAccessToken, and this function uses `map` to compose, instead of having a case per error